pkg/render/policyrecommendation_test.go

```diff
@@ -321,6 +326,7 @@ var _ = Describe("Policy recommendation rendering tests", func() {
 					ID: "tenant-b-id",
 				},
 			}
+			cfg.BindingNamespaces = []string{tenantBNamespace}
```

Comment:
We should include both namespaces here so that we properly verify the binding includes two subjects.